src/openrct2/world/Sprite.cpp

```diff
@@ -537,17 +542,29 @@ void sprite_set_coordinates(const CoordsXYZ& spritePos, SpriteBase* sprite)
     sprite->z = spritePos.z;
 }
 
+static void DestroyEntity(SpriteBase* entity)
```

Comment:
This is in essence the same name as `sprite_remove`. It’s also a very non-descriptive name, as it doesn’t actually destroy the entity or anything close to it.

Reply:
It's part of sprite remove and the naming is from the nsf branch where it also destroys the staff patrol area. I can rename it but will cause merge with nsf to be more annoying.

Reply:
OK, can you create an issue to change it, then?

Reply:
Seems the nsf had a different name so ive swapped to that name `FreeEntity`